tree/dataframe/src/RDataFrame.cxx

```diff
@@ -629,6 +629,16 @@ df = RDataFrame("mytree", "myfile.root", sparkcontext = sc)
 If an instance of [SparkContext](https://spark.apache.org/docs/latest/api/python/reference/api/pyspark.SparkContext.html)
 is not provided, the default behaviour is to create one in the background for you.
 
+### Distributed Snapshot
+
+The Snapshot operation behaves slightly differently when executed distributedly. First off, it requires the path
+supplied to the Snapshot call to be accessible from any worker of the cluster and from the client machine (in general
+it should be provided as an absolute path). Then, each distributed task will make a Snapshot on the range of entries it
+has been assigned to. Thus, there will be `n` separate files at the location provided (with `n` being the number of 
+ranges), each containing the result of the Snapshot call on its processed range. As with local RDataFrame, the result of
```

Comment:
even less discussion of implementation details:

```suggestion
it should be provided as an absolute path). Another important difference is that `n` separate files will be produced, where `n` is the number of dataset partitions. As with local RDataFrame, the result of
```

Reply:
Done in rebased commit